lib/rules/no-restricted-imports.js

```diff
@@ -229,16 +254,32 @@ module.exports = {
                             loc: specifierData.loc,
                             data: {
                                 importSource,
-                                importNames: restrictedImportNames,
+                                importNames: restrictedImportNames || ["*"],
```

Comment:
this would result something like `"* import is invalid because '*' from 'foo' is restricted` this can lead confusion and also on importing `*` from a module now we are getting two errors instead of one like this
`* import is invalid because '*' from 'foo' is restricted`
`'*' import from 'foo' is restricted`

Reply:
@Tanujkanti4441 Look interesting! Would you like to apply and test your changes please? If you create a pull request I will update it.

Reply:
I have tested these changes you can see them in PR #18086

Reply:
@Tanujkanti4441 Thank you for your PR,, although I meant a PR derived from and into this branch please :https://github.com/MichielPater/eslint/tree/feat-no-restricted-imports-excludeImportNames

Reply:
@Tanujkanti4441 Would you send a pull request to my branch please?

Reply:
I have [PR](https://github.com/MichielPater/eslint/pull/1)

Reply:
@Tanujkanti4441 Merged, thank you.